pkg/storage/store.go

```diff
@@ -68,6 +74,9 @@ type Store interface {
 	// Query queries the store for data based on the provided query string, the format of
 	// which will be dependent on what the underlying store requires.
 	Query(query string) (ResultsIterator, error)
+
+	// Delete deletes the key-value pair associated with k.
+	Delete(k string) error
```

Comment:
I'm wondering if `Remove` would make more sense as other APIs have Put/Get rather than Insert/Select.

Reply:
Sounds reasonable to me...done!

Reply:
This change makes no sense and I wouldn't be surprised if it is reverted in the future:

* redis: https://redis.io/commands/del
* memcache: https://www.tutorialspoint.com/memcached/memcached_delete_key.htm
* aries-framework-go's `Store` API
* CouchDB: https://docs.couchdb.org/en/stable/api/document/common.html#delete--db-docid

Reply:
MySQL also uses `DELETE` (as can be seen in the code added here)... so it looks to me like the original "Delete" terminology was the best choice.

(it looks like some APIs may combine Put/Get with "Remove", but after a closer look I think this isn't universal...)

Reverted.